framework/wazuh/core/utils.py

```diff
@@ -1032,3 +1033,60 @@ def get_utc_strptime(date: str, datetime_format: str) -> datetime:
         The current date.
     """
     return datetime.strptime(date, datetime_format).replace(tzinfo=timezone.utc)
+
+
+class KeystoreReader:
+    """Class to read values from a keystore."""
+
+    KEY_SIZE = 32
+    IV_SIZE = 16
+    KEY_VALUE_SEPARATOR = ':'
+
+    def __init__(self, keystore_path: Path):
+        self._data = keystore_path.read_bytes()
+        self._cipher = self._get_cipher()
+        self._keystore = self._decrypt_keystore()
+
+    @staticmethod
+    def _unpad(decrypted):
+        return decrypted[: -ord(decrypted[len(decrypted) - 1 :])]
+
+    def _get_cipher(self):
+        enc_key = self._data[: self.KEY_SIZE]
+        iv = self._data[self.KEY_SIZE : self.KEY_SIZE + self.IV_SIZE]
+        return AES.new(enc_key, AES.MODE_CBC, iv)
+
+    def _decrypt_keystore(self) -> dict:
+        value = self._data[self.KEY_SIZE + self.IV_SIZE :]
+        decrypted = self._unpad(self._cipher.decrypt(value)).decode('utf-8')
+
+        lines = decrypted.split('\n')
+        lines.pop()
+        kev_values = {}
+        for line in lines:
+            key_value = line.split(self.KEY_VALUE_SEPARATOR)
+            key = key_value[0]
+            value = key_value[1]
+            kev_values[key] = value
+
+        return kev_values
+
+    def get(self, key: str, default: typing.Optional[str] = None) -> typing.Optional[str]:
```

Comment:
We can avoid using the `typing` module for `Optional` type. https://docs.python.org/3/library/typing.html#typing.Optional

```suggestion
    def get(self, key: str, default: str | None) -> str | None:
```